components/script/dom/globalscope.rs

```diff
@@ -2227,6 +2231,66 @@ impl GlobalScope {
         }))
     }
 
+    pub fn create_image_bitmap(
+        &self,
+        image: ImageBitmapSource,
+        options: &ImageBitmapOptions,
+    ) -> Rc<Promise> {
+        let global = self.global();
+        let in_realm_proof = AlreadyInRealm::assert(&global);
+        let p = Promise::new_in_current_realm(&global, InRealm::Already(&in_realm_proof));
+        if options.resizeWidth.unwrap() == 0 {
+            p.reject_error(Error::InvalidState)
+        }
+
+        if options.resizeHeight.unwrap() == 0 {
+            p.reject_error(Error::InvalidState)
```

Comment:
These early errors should return p immediately to avoid running more code in an error state.

Reply:
got it